test/test_quantized_tensor.py

```diff
@@ -77,13 +77,15 @@ def test_per_channel_qtensor_creation(self):
         q = torch._empty_per_channel_affine_quantized_like(scales, zero_points, [numel], [ch_axis], dtype=torch.quint8)
         self.assertEqual(scales, q.q_per_channel_scales())
         self.assertEqual(zero_points, q.q_per_channel_zero_points())
+        self.assertEqual([ch_axis], q.q_per_channel_axis())
```

Comment:
Both per_channel and qtensor creation tests should sweep over symmetric and affine quant.